valueformatting_test.go

```diff
@@ -520,6 +544,41 @@ func TestValueFormattingFormat(t *testing.T) {
 	}
 }
 
+func TestJSONAndYAML(t *testing.T) {
+	globalValueFormatting = newValueFormatting()
+	globalValueFormatting.setup(filepath.Join("testdata", "cat.yml"))
+
+	row := bigtable.Row{
+		"f1": {
+			bigtable.ReadItem{
+				Row:    "r1",
+				Column: "f1:json",
+				Value:  []byte("{\"name\": \"Brave\", \"age\": 2}"),
+			},
+		},
+	}
+	var out bytes.Buffer
+
+	printRow(row, &out)
+	got := out.String()
+	want := ("----------------------------------------\n" +
+		"r1\n" +
+		"  f1:json\n" +
+		"    age:   2.00\n" +
+		"    name: \"Brave\"")
+
+	timestampsRE := regexp.MustCompile("[ ]+@ [^ \t\n]+")
+
+	stripTimestamps := func(s string) string {
+		return string(timestampsRE.ReplaceAll([]byte(s), []byte("")))
+	}
+	got = stripTimestamps(got)
```

Comment:
We can probably inline this function as `ReplaceAll` is pretty clear about what it's doing.

Reply:
Done.